config/versions/v1/v1.go

```diff
@@ -1,26 +1,21 @@
-package versions
+package v1
 
 import (
 	"context"
 
 	"github.com/buger/jsonparser"
 	v0 "github.com/thrasher-corp/gocryptotrader/config/versions/v0"
-	v1 "github.com/thrasher-corp/gocryptotrader/config/versions/v1"
 	"github.com/thrasher-corp/gocryptotrader/encoding/json"
 )
 
-// Version1 is an ExchangeVersion to upgrade currency pair format for exchanges
-type Version1 struct{}
-
-func init() {
-	Manager.registerVersion(1, &Version1{})
-}
+// Version is an ExchangeVersion to upgrade currency pair format for exchanges
+type Version struct{}
 
 // Exchanges returns all exchanges: "*"
-func (v *Version1) Exchanges() []string { return []string{"*"} }
+func (v *Version) Exchanges() []string { return []string{"*"} }
```

Comment:
You dont need the method receiver across these methods in the versions can just be `(v *Version)` -> `(Version)`

Reply:
Yep.

I'm going to leave it as a pointer method, though.

Fixed https://github.com/gbjk/gocryptotrader/commit/4fec6cd435